examples/multi-alignment.rs

```diff
@@ -0,0 +1,299 @@
+use std::time::Duration;
```

Comment:
Instead of adding yet another example, can we change one of the existing ones to switch alignment based on a command-line flag? I think there are already too many examples and we probably shouldn't add any at this point (and this seems like a fairly niche feature).

Reply:
Thanks for review! I have extended `multi-tree-ext` example instead.